traduki/sqla.py

```diff
@@ -112,11 +112,15 @@ def contains(self, other, escape=None):
 
         def _do_compare(self, op, other, escape):
             """Perform coalesced comparison operations to the columns of Translation model.
```

Comment:
It is not a coalesced comparison anymore

Reply:
Done.